grammars/pronounce.grm

```diff
@@ -270,10 +270,10 @@ test_pron_18 = AssertEqual[
     "prajdãː adserwaːbant huːk undikwe troːia gazza"
 ];
 
-# Tests "au" diphthongization.
+# Tests that nasalization occurs before au diphthonization.
 test_pron_19 = AssertEqual[
     "aut pelagō danaum īnsidiās suspectaque dōna" @ PRONOUNCE,
-    "awt pelagoː danawm iːnsidiaːs suspektakwe doːna"
+    "awt pelagoː danaũː iːnsidiaːs suspektakwe doːna"
```

Comment:
Maybe a silly question but how do we know this is right? I wouldn't have guessed you could do this.

Reply:
The "ũː" is supposed to elide into the next vowel, but if it was "danawn" it wouldn't be able to

Here is the textproto verse:
  ```
number: 37
  text: "aut pelagō Danaum īnsidiās suspectaque dōna"
  norm: "aut pelagō danaum īnsidiās suspectaque dōna"
  raw_pron: "awt pelagoː danaũː iːnsidiaːs suspektakwe doːna"
  var_pron: "awt pelagoː danaiːnsidiaːs suspektakwe doːna"
```

Reply:
Do we have other evidence of, specifically, the second, but not the first, half of a diphthong undergoing both elision? I would have expected the entire <au> to delete, if I had to guess.

Dumb question but shouldn't it also be [ĩːsidiaːs] because of this rule? https://github.com/CUNY-CL/latin_scansion/blob/cc124b49724ecfaa26d001d7be49f0722b90e756/grammars/pronounce.grm#L122

Reply:
I don't think we have encountered other examples of it, since it's a relatively specific occurrence, i.e. word final m preceded by a diphthong. Word final diphthongs would elide in its entirety, but here it seems like the elision would supersede the formation of the diphthong. Any other word final consonant that follows a diphthong wouldn't be able to elide. Would you agree with this analysis?

Reply:
> Dumb question but shouldn't it also be [ĩːsidiaːs] because of this rule?
> 
> https://github.com/CUNY-CL/latin_scansion/blob/cc124b49724ecfaa26d001d7be49f0722b90e756/grammars/pronounce.grm#L122

Yeah, looking at it, it has to do with the way the rules are ordered. I could move the `@ internal_nasalization` rule up ahead, although that would mess up neatness of the organization of rules if that is okay with you.

Reply:
Thanks for thinking through this a bit; let me sleep on the analysis just a bit, if that's okay.

Yeah, maybe we should move that nasalization rule higher, since that is a minor error; could you do that?

Reply:
Do you have new opinions on this?